WcaOnRails/app/models/user.rb

```diff
@@ -1081,6 +1081,7 @@ def to_wcif(competition, registration = nil, registrant_id = nil, authorized: fa
     person_pb = [person&.ranksAverage, person&.ranksSingle].compact.flatten
     roles = registration&.roles || []
     roles << "delegate" if competition.delegates.include?(self)
+    roles << "trainee_delegate" if competition.trainee_delegates.include?(self)
```

Comment:
```suggestion
    roles << "trainee-delegate" if competition.trainee_delegates.include?(self)
```

Reply:
Could you elaborate on why to use a hyphen instead of underscore? I used an underscore as this was how Trainees were implemented in this PR https://github.com/thewca/worldcubeassociation.org/pull/5249

Reply:
See https://github.com/thewca/wcif/pull/4#discussion_r805412204, it's just to align with the conventions in WCIF, since that's what we are building here :)

In all other places underscores are fine as they follow the conventions of Ruby code.

Reply:
Simply put, there is a difference between what we do within our Ruby code (underscore) and what we publicly expose to the outside world (cleanly defined WCIF spec, which happens to use hyphens)

Reply:
Alright, thanks! Should all be good now then.